client/my-sites/activity/activity-log/index.jsx

```diff
@@ -605,6 +616,7 @@ export default connect(
 			timezone,
 			siteHasNoLog,
 			isIntroDismissed: getPreference( state, 'dismissible-card-activity-introduction-banner' ),
+			supportsRealtimeBackup: siteSupportsRealtimeBackup( state, siteId ),
```

Comment:
Like noted in one of the following comments - I think it would be useful to note in a comment that this selector needs `QuerySitePurchases` to work.

Reply:
See https://github.com/Automattic/wp-calypso/pull/38473#discussion_r362475829